tests/key_field/key_field_sample_generated.h

```diff
@@ -114,14 +116,64 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(4) Bar FLATBUFFERS_FINAL_CLASS {
 };
 FLATBUFFERS_STRUCT_END(Bar, 16);
 
+FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(4) BarParent FLATBUFFERS_FINAL_CLASS {
+ private:
+  uint8_t a_;
+  int8_t padding0__;  int16_t padding1__;
+  keyfield::sample::Bar b_;
+
+ public:
+  BarParent()
+      : a_(0),
+        padding0__(0),
+        padding1__(0),
+        b_() {
+    (void)padding0__;
+    (void)padding1__;
+  }
+  BarParent(uint8_t _a, const keyfield::sample::Bar &_b)
+      : a_(flatbuffers::EndianScalar(_a)),
+        padding0__(0),
+        padding1__(0),
+        b_(_b) {
+    (void)padding0__;
+    (void)padding1__;
+  }
+  uint8_t a() const {
+    return flatbuffers::EndianScalar(a_);
+  }
+  const keyfield::sample::Bar &b() const {
+    return b_;
+  }
+  bool KeyCompareLessThan(const BarParent * const o) const {
```

Comment:
I manually changed this KeyCompareLessThan method to first convert the struct to an uint8_t array and then compare the array, but it is not working correct for negative numbers.